spec/features/swagger_spec.rb

```diff
@@ -49,8 +49,8 @@
       it 'adds an Authorization header' do
```

Comment:
I'd like additional tests for this rather than editing existing tests. I am not sure how possible/easy it is because there's this global `api_auth` and `api_key_type` setup.